cmd/tempo-cli/cmd-list-block.go

```diff
@@ -59,7 +59,15 @@ func dumpBlock(r tempodb_backend.Reader, c tempodb_backend.Compactor, tenantID s
 	if checkDupes {
 		fmt.Println("Searching for dupes ...")
 
-		iter, err := encoding.NewBackendIterator(tenantID, id, 10*1024*1024, r)
+		block, err := encoding.NewBackendBlock(&tempodb_backend.BlockMeta{
+			TenantID: tenantID,
+			BlockID:  id,
```

Comment:
This is missing the version.  The original meta is available above, and it probably makes sense to add to the unifiedMeta and display it.

Reply:
good catch.